qfieldsync/gui/cloud_projects_dialog.py

```diff
@@ -555,7 +557,9 @@ def show_projects(self) -> None:
 
         if len(self.network_manager.projects_cache.projects) == 0:
             self.set_feedback(
-                "You don't have any projects, create some by clicking the button in the bottom bar.",
+                self.tr(
+                    "You don't have any projects, create some by clicking the button in the bottom bar."
```

Comment:
```suggestion
                    "You don't have any projects, create one by clicking the button in the bottom bar."
```

Let's start with a single one 😅